module4/ex10_wedding_planner/ex10_wedding_planner.py

```diff
@@ -1,10 +1,75 @@
-class GuestList:
+from collections import namedtuple
+
+Person = namedtuple('Person', ['first', 'last'])
+
+
+class TableFull(Exception):
+    """Raised when """
     pass
 
 
+class GuestList:
+    max_at_table = 10
+
+    def __init__(self):
+        self.persons = {}
+        self.tables = {}
+
+    def __len__(self):
+        return len(self.persons.keys())
+
+    def add_to_table(self, person, table_number):
+        if table_number:
+            if table_number in self.tables.keys():
```

Comment:
It is enough to just write `if table_number in self.tables:` because `in` tests for the existence of a key in a dictionary.